pkg/api/meta/meta.go

```diff
@@ -30,14 +30,14 @@ import (
 )
 
 func ListAccessor(obj interface{}) (List, error) {
-	if oi, ok := obj.(ListMetaAccessor); ok {
-		if om := oi.GetListMeta(); om != nil {
+	if listMetaAccessor, ok := obj.(ListMetaAccessor); ok {
+		if om := listMetaAccessor.GetListMeta(); om != nil {
 			return om, nil
 		}
 	}
 	// we may get passed an object that is directly portable to List
-	if oi, ok := obj.(List); ok {
-		return oi, nil
+	if list, ok := obj.(List); ok {
```

Comment:
Check for this first, since the object could support this and the list meta accessor?